subsys/usb/class/netusb/netusb.c

```diff
@@ -57,6 +57,11 @@ static int netusb_send(struct device *dev, struct net_pkt *pkt)
 	return 0;
 }
 
+struct net_iface *netusb_net_iface(void)
+{
+	return netusb.iface;
+}
+
```

Comment:
This doesn't compile, `netusb.iface` is of type `struct net_if *` while the function tries to return a `struct net_iface *`.

Reply:
oh yes...